app/schemas/FOOD.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "foodId": {
+      "type": "string"
+    },
+    "name": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "nutritionalData": {
+      "type": "object",
+      "properties": {
+        "calories": {
+          "type": "integer"
+        },
+        "protein": {
+          "type": "string"
+        },
+        "carbs": {
+          "type": "string"
+        },
+        "fats": {
+          "type": "string"
+        },
+        "fiber": {
+          "type": "string"
+        }
+      },
+      "required": ["calories", "protein", "carbs", "fats", "fiber"]
+    },
+    "menuId": {
+      "type": "string"
+    }
+  },
+  "required": ["foodId", "name", "description", "nutritionalData", "menuId"]
```

Comment:
Nice! In the markdown table, it is possible to make a column that signifies which fields are required for developer quick reference? I don't think firebase will check any of this for us, so as a teams of devs we need to be really clear about the data we're sending to the backend.